packages/utils/telemetry-utils/src/test/logger.spec.ts

```diff
@@ -123,13 +123,6 @@ describe("Logger", () => {
                 assert(!(event.stack as string).includes("boom"));
             });
         });
-        describe("TaggedTelemetryData", () => {
-            it("Ensure backwards compatibility", () => {
-                // The values of the enum should never change (even if the keys are renamed)
-                assert(TelemetryDataTag.PackageData === "PackageData" as TelemetryDataTag);
-                assert(TelemetryDataTag.UserData === "UserData" as TelemetryDataTag);
-            });
-        });
```

Comment:
This should stay.  These strings are extremely important to keep as stable as possible.

Reply:
added back